src/pal/inc/rt/xmmintrin.h

```diff
@@ -27,12 +27,16 @@
 *===-----------------------------------------------------------------------===
 */
 
-#ifdef __clang__
+#ifdef __GNUC__
 
 typedef float __m128 __attribute__((__vector_size__(16)));
 
 /* Define the default attributes for the functions in this file. */
+#ifdef __clang__
 #define __DEFAULT_FN_ATTRS __attribute__((__always_inline__, __nodebug__))
```

Comment:
I've found that GCC has an attribute `__artificial__` that serves the same purpose as `__nodebug__`.  It seems to be recognized by GCC from 4.4 on.

Reply:
Thanks for the hint, let me rework.